test/e2e/scripts/setup-gitlab.sh

```
@@ -643,7 +643,53 @@ if command -v docker >/dev/null 2>&1; then
+            --data-urlencode "skip_confirmation=true" \
+            --connect-timeout 5 --max-time 30 2>/dev/null || true)
+        pending_id=$(json_field "${pending_json}" "id")
+        if [ -z "${pending_id}" ]; then
+            echo "      WARNING: could not create pending-${role} user; approve/reject happy path will skip"
+            pending_ids=""
+            break
+        fi
+        pending_ids="${pending_ids} ${pending_id}"
+    done
+    if [ -n "${pending_ids}" ]; then
+        pending_ids_csv=$(echo ${pending_ids} | tr ' ' ',')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)